### ISSUE TITLE
[Question] - Is there any plan to enhance roosterjs-react using content model apis in @fluentui/react-component (V9)

### ISSUE THREAD

**Issue by pradeept95** [microsoft/roosterjs#2567]
Currently, the react example is using components from "@fluentui/react (V8)", is there any plan to enhance this using "@fluentui/react-component (V9)"?

**Comment by JiuqingSong** [comment]
No. Because roosterjs-react is mostly used for demo and build quick prototype. We won't spend too much time on it.

**Comment by LarsKemmann** [comment]
So, what are people with an interest in this library supposed to do if they're using React (which would be _everyone_, inside and outside of MSFT, who's using Fluent UI v9), @JiuqingSong? At a minimum, provide some documentation (see https://github.com/microsoft/roosterjs/issues/2618).

**Comment by pradeept95** [comment]
Hi @LarsKemmann , this might help: https://github.com/microsoft/roosterjs/issues/2618#issuecomment-2108539165
